scalafmt-core/shared/src/main/scala/org/scalafmt/config/Indents.scala

```diff
@@ -13,6 +14,7 @@ import metaconfig._
   */
 case class Indents(
     main: Int = 2,
+    significant: Int = 3,
```

Comment:
I would just use  `main` this way the diffs will be minimal if people switch to 3 and start using it.

Reply:
you mean:
```
significant: Int = 2
```
or
```
significant: Option[Int] = None
```
followed by
```
def getSignificant = significant.getOrElse(main)
```

Reply:
I would just use main and not add a new setting.

Reply:
oh... i didn't expect that :) i thought you said in another discussion that martin recommended 3 for significant indentation...

Reply:
That's the recommendation, but we were far from consensus there. If we do default of 2 then at least when people drop optional braces it will not cause additional changes. They can change to 3 or 4 if they want to.